src/widget/wtrackmenu.cpp

```diff
@@ -1377,6 +1402,20 @@ void WTrackMenu::slotReanalyze() {
     addToAnalysis();
 }
 
+void WTrackMenu::slotReanalyzeWithFixedTempo() {
+    clearBeats();
+    AnalyzerTrack::Options options;
+    options.useFixedTempo = true;
```

Comment:
I think the API would be more future prove to have here also the option "from preferences" or such.